[tree view] Multi character type-ahead (#19942)
mui/mui-x: docs/data/tree-view/accessibility/accessibility.md

|                                                                    <kbd class="key">End</kbd> | Focuses the last item in the tree                                                                                                                                                                                    |
|                                                                     <kbd class="key">\*</kbd> | Expands all siblings that are at the same level as the focused item without moving focus                                                                                                                             |

> Type-ahead is supported for single or multiple characters. When typing a character, focus moves to the next item with a label that starts with the typed characters.

Author: WDYT about:

```suggestion
When typing a set of characters, focus moves to the next item with a label that starts with the typed characters.
```
